stripe.go

```diff
@@ -348,3 +382,24 @@ func (s *BackendConfiguration) ResponseToError(res *http.Response, resBody []byt
 
 	return stripeErr
 }
+
+// getUname tries to get a uname from the system, but not that hard. It tries
+// to execute `uname -a`, but swallows any errors in case that didn't work
+// (i.e. non-Unix non-Mac system or some other reason).
+func getUname() string {
+	path, err := exec.LookPath("uname")
+	if err != nil {
+		return UnknownPlatform
+	}
+
+	cmd := exec.Command(path, "-a")
+	var out bytes.Buffer
+	cmd.Stderr = nil // goes to os.DevNull
+	cmd.Stdout = &out
+	err = cmd.Run()
+	if err != nil {
+		return UnknownPlatform
+	}
+
+	return out.String()
+}
```

Comment:
So one note here: the way we're trying to fetch unames across libraries isn't consistent. Where the language has a built-in function (PHP, Python) we usually use that. stripe-ruby does something a little special in that it tries extra hard to get a uname based on a whole bunch of different switches. I didn't really want to reimplement that, so I'm just using the stripe-node technique of trying to run `uname -a`.

I think this should generally be suitable because it would be pretty surprising for people to be running server side Go on non-Unix non-Mac OS systems (I'm sure it's done somewhere, but it's probably very unusual).